packages/desktop-gui/src/runs/setup-project-modal.jsx

```diff
@@ -157,43 +157,48 @@ class SetupProject extends Component {
-              })}>
-                <input
-                  type='radio'
-                  name='owner-toggle'
-                  id='me'
-                  autoComplete='off'
-                  value='me'
-                  checked={this.state.owner === 'me'}
-                  onChange={this._updateOwner}
-                />
-                <img
-                  className='user-avatar'
-                  height='13'
-                  width='13'
-                  src={`${gravatarUrl(authStore.user && authStore.user.email)}`}
-                />
-                {' '}Me
-              </label>
-              <label className={`btn btn-default ${this.state.owner === 'org' ? 'active' : ''}`}>
-                <input
-                  type='radio'
-                  name='owner-toggle'
-                  id='org'
-                  autoComplete='off'
-                  value='org'
-                  checked={this.state.owner === 'org'}
-                  onChange={this._updateOwner}
-                />
-                <i className='far fa-building'></i>
-                {' '}An Organization
-              </label>
-            </div>
-          </div>
+          {
+            this._hasDefaultOrg() ?
+              <div>
+                <div className='btn-group' data-toggle='buttons'>
+
+                  <label className={cs('btn btn-default', {
+                    'active': this.state.owner === 'me',
+                  })}>
+                    <input
+                      type='radio'
+                      name='owner-toggle'
+                      id='me'
+                      autoComplete='off'
+                      value='me'
+                      checked={this.state.owner === 'me'}
+                      onChange={this._updateOwner}
+                    />
+                    <img
+                      className='user-avatar'
+                      height='13'
+                      width='13'
+                      src={`${gravatarUrl(authStore.user && authStore.user.email)}`}
+                    />
+                    {' '}Me
+                  </label>
+                  <label className={`btn btn-default ${this.state.owner === 'org' ? 'active' : ''}`}>
+                    <input
+                      type='radio'
+                      name='owner-toggle'
+                      id='org'
+                      autoComplete='off'
+                      value='org'
+                      checked={this.state.owner === 'org'}
+                      onChange={this._updateOwner}
+                    />
+                    <i className='far fa-building'></i>
+                    {' '}An Organization
+                  </label>
+                </div>
+              </div> :
+              null
+          }
           <div className='select-orgs'>
             <div className={cs({ 'hidden': this.state.owner !== 'org' || this._hasOrgsOtherThanDefault() })}>
               <div className='empty-select-orgs well'>
```

Comment:
This error should probably be updated. The Dashboard refers to the default org as "Your personal organization", but it will still say "You don't have any organizations yet" if you only have a personal org:

![image](https://user-images.githubusercontent.com/1151760/70824401-317dd080-1db0-11ea-9a29-ea53cb72be89.png)

![image](https://user-images.githubusercontent.com/1151760/70824477-5a05ca80-1db0-11ea-8a1e-81624efc742b.png)

Reply:
I couldn't comment on the correct lines of code because they aren't edited in this PR (yet) but it's just below where I commented